projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -527,6 +527,23 @@ public function get_display_name( $service_name, $connection ) {
 		return $connection_display;
 	}
 
+	/**
+	 * Returns the account name for the Connection. For services like Instagram, we need both the username and the account's name.
+	 *
+	 * @param string       $service_name 'facebook', 'linkedin', etc.
+	 * @param object|array $connection The Connection object (WordPress.com) or array (Jetpack).
+	 * @return string
+	 */
+	public function get_account_display_name( $service_name, $connection ) {
```

Comment:
It might be more consistent to not mention `display` here, and if this is the username, then we should probably rename this to `get_username`